frontend/micro-ui/web/micro-ui-internals/packages/Mukta/src/configs/UICustomizations.js

```diff
@@ -1586,18 +1587,37 @@ export const UICustomizations = {
           "order": "ASC"
         }
       })
-
-      const payload = getCreatePaymentPayload(result.bills);
+      
```

Comment:
create a new method for this payment advice create 
if someone goes with previous feature let them go

Reply:
Updated @jagankumar-egov